setup.py

```diff
@@ -16,13 +16,10 @@
 import sys
 import subprocess
 import platform
-from distutils.sysconfig import get_config_var
-from distutils.version import LooseVersion
+from sysconfig import get_config_var
```

Comment:
I've never used this before. So `sysconfig` is all about getting install-level configuration settings of Python, not environment variables? Sounds good then.

Reply:
From the [`sysconfig` documentation](https://docs.python.org/3/library/sysconfig.html):
> A Python distribution contains a `Makefile` and a `pyconfig.h` header file that are necessary to build both the Python binary itself and third-party C extensions compiled using [`distutils`](https://docs.python.org/3/library/distutils.html#module-distutils).
> 
> [`sysconfig`](https://docs.python.org/3/library/sysconfig.html#module-sysconfig) puts all variables found in these files in a dictionary that can be accessed using [`get_config_vars()`](https://docs.python.org/3/library/sysconfig.html#sysconfig.get_config_vars) or [`get_config_var()`](https://docs.python.org/3/library/sysconfig.html#sysconfig.get_config_var).

[`sysconfig.get_path()`](https://docs.python.org/3/library/sysconfig.html#sysconfig.get_path) is something I've used to get the module path for example:
```python
>>> sysconfig.get_path('purelib')
'/usr/local/lib/python3.11/dist-packages'
>>> sysconfig.get_path('purelib', 'deb_system')
'/usr/lib/python3/dist-packages'
```